src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -121,17 +123,39 @@ const formConfig = {
         }
       }
     },
-    chapterThree: {
-      title: 'Chapter Three',
+    ratedDisabilities: {
+      title: 'Your Rated Disabilities',
       pages: {
-        pageOne: {
-          title: 'Page One',
-          path: 'chapter-three/page-one',
-          uiSchema: {},
+        ratedDisabilities: {
+          title: 'Your Rated Disabilities',
+          path: 'select-disabilities',
+          uiSchema: {
+            'view:selectDisabilitiesDescription': {
+              'ui:description': 'Please check all the disabilities that you’re filing an increase for because the condition has gotten worse.'
+            },
+            disabilities: {
+              // Using StringField because it doesn't do much and we just need to render the widget.
+              // If this becomes a common(ish) pattern, we should make a BasicField or something.
+              'ui:field': 'StringField',
+              'ui:widget': 'SelectArrayItemsWidget',
+              'ui:options': {
```

Comment:
For validation, I think you can add a validation function in `ui:validations` here that adds an error if none of the items is selected.

Reply:
Ahh, okay. I'll give that a shot, thanks!